pkg/remote/remote.go

```diff
@@ -154,10 +154,14 @@ func authOptions(ctx context.Context, ref name.Reference, option types.RegistryO
 		}))
 	}
 
-	domain := ref.Context().RegistryStr()
-	token := registry.GetToken(ctx, domain, option)
-	if !lo.IsEmpty(token) {
-		opts = append(opts, remote.WithAuth(&token))
+	// The default trivy-db and trivy-java-db repositories are public.
```

Comment:
If I understand correctly, there is no chance that `mirror.gcr.io` requires authentication. We should skip `mirror.gcr.io` here.
https://github.com/aquasecurity/trivy/blob/94791f83431c02a566f73e4e3f00557d3e90aef5/pkg/fanal/image/registry/google/google.go#L32

Reply:
yeah, you're probably right, I didn't think about it
Changed in https://github.com/aquasecurity/trivy/pull/7679/commits/99e96bbd53777740fc0957211438ae336fff72d3